frontend/public/components/build-config.tsx

```diff
@@ -65,8 +65,36 @@ const startBuildAction: KebabAction = (kind, buildConfig) => ({
   },
 });
 
+const isLastbuildPresent = (build) => (build ? true : false);
+
+const startLastBuildAction: KebabAction = (kind, buildConfig: BuildConfig) => {
+  return {
+    // t('public~Start last run')
+    labelKey: 'public~Start last run',
+    callback: () =>
+      cloneBuild(buildConfig.latestBuild)
+        .then((clone) => {
+          history.push(resourceObjPath(clone, referenceFor(clone)));
+        })
+        .catch((err) => {
+          const error = err.message;
+          errorModal({ error });
+        }),
+    hidden: !isLastbuildPresent(buildConfig.latestBuild),
+    accessReview: {
+      group: kind.apiGroup,
+      resource: kind.plural,
+      subresource: 'instantiate',
+      name: buildConfig.metadata.name,
+      namespace: buildConfig.metadata.namespace,
+      verb: 'create',
+    },
+  };
+};
+
 const menuActions = [
   startBuildAction,
+  startLastBuildAction,
```

Comment:
I can not see this action in my BuildConfig action menu :thinking: 

![image](https://github.com/openshift/console/assets/139310/25d95e4f-b0f4-4ca0-b30e-b26d2118c82d)